website/docs/reference/resource-configs/grants.md

```diff
@@ -98,32 +113,109 @@ See [configs and properties](configs-and-properties) for details.
 </TabItem>
 </Tabs>
 
-## Definition
+### Grant config inheritance
 
-You can use the `grants` field to set permissions or grants for a resource. These grants will be compiled into the `manifest.json` file complied by dbt.
+If `grants` are set for the same model in multiple places, such as in `dbt_project.yml` and in a more-specific `.sql` or `.yml` file, the default behavior is to "merge and clobber" each privilege. That is, dbt will replace the less-specific set of grantees with the more-specific set of grantees.
 
-## Database-specific requirements and notes
+For example:
 
-While we try to standardize the terms we use to describe different features, you will always find nuances in different databases. This section outlines some of those database-specific requirements and notes.
+<File name='dbt_project.yml'>
 
-### Common syntax 
+```yml
+models:
+  +grants:
+    select: ['user_a', 'user_b']
+```
+
+</File>
+
+<File name='models/specific_model.sql'>
+
+```sql
+{{ config(grants = {'select': ['user_c']}) }}
+```
+
+</File>
+
+As a result of this configuration, `specific_model` will grant the `select` privilege to `user_c` ONLY.
+
+Let's say we wanted to _add_ `user_c` to the existing list of grantees receiving the `select` privilege on `specific_model`, rather than _replacing_ that list entirely. To accomplish that, we can use the `+` ("addition") symbol, prefixing the name of the privilege:
+
+<File name='models/specific_model.sql'>
+
+```sql
+{{ config(grants = {'+select': ['user_c']}) }}
+```
+
+</File>
 
-In our examples, you will find terms like `select` and `another_user` because many databases use these terms, but be aware of the syntax your own database supports:
+Now, the model will grant select to `user_a`, `user_b`, AND `user_c`!
 
-* Privileges: A right to perform an action in a database.
-* Grantees: A way to manage privileges. Recipients of granted privileges, also called "principals." Grantees can be a user, a group of users, a role held by users (Snowflake), a service account (GCP), and more.
+**Notes:**
+- This will only take effect for privileges which include the `+` prefix. Each privilege controls that behavior separately. If we were granting other privileges, in addition to `select`, and those privilege names lacked the `+` prefix, they would continue to "clobber" rather than "add" new grantees.
+- This use of `+`, controlling clobber vs. add merge behavior, is distinct from the use of `+` in `dbt_project.yml` (shown in the example above) for defining configs with dictionary values. For more information, see [the plus prefix](https://docs.getdbt.com/reference/resource-configs/plus-prefix).
```

Comment:
🤢

Reply:
I know :(

the other `+` prefix is the worse one IMO. I wanted to (and still want to) replace it with `config:`, for consistency across the board: https://github.com/dbt-labs/dbt-core/issues/3663